src/main/java/dao/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void write(String content, String filePath) {
```

Comment:
According to the checklist, avoid hardcoding the path to resources in your service implementation. Instead, the path should be passed as a parameter to the method, which you have correctly implemented here. However, ensure that the path provided is always a relative path and not an absolute path for it to work on any system.